kuma/wiki/views/misc.py

```diff
@@ -157,6 +160,7 @@ def helpful_vote(request, document_path):
     return redirect(document)
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/load/ to robots.txt? (superuser only)